tests/testthat/test-template.R

```diff
@@ -46,6 +46,27 @@ test_that("UTF-8 characters in templates", {
   expect_identical(charToRaw(text), as.raw(c(0xc3, 0xbf)))
 })
 
+test_that("UTF-8 characters in template head but not body", {
+  # On Windows, a string with "中文" will automatically be marked as UTF-8.
+  ui <- tagList(
+    tags$head(tags$script("alert('中文')")),
```

Comment:
I just realized these tests might fail on Windows. I'm not sure if R can read these tests with the correct encoding, especially when the system locale is not Chinese. We may want to `skip_on_cran()`.